napari/layers/image/_tests/test_image.py

```diff
@@ -4,7 +4,7 @@
 import xarray as xr
 
 from napari._tests.utils import check_layer_world_data_extent
-from napari.layers import Image
+from napari.layers import Image, Labels
```

Comment:
I don't love that we are importing the Labels in the Image test - maybe this reflects a poor design choice, but I had liked keeping the tests for the different layers in their own folders. I guess it makes it hard to do parametric tests like bellow. We do have some tests in `layers/_tests` that are parametric. Curious what @jni and @tlambert03 think here. I'm happy to be overruled, but wanted to point it out

Reply:
Yup, I'd move this to `layers/_tests`, though I'm not too worried. Alternatively, we could have the layers folder organisation reflect our class hierarchy — so we would have this somewhere like layers/image_base/_tests. If we want to go that route, I would leave that reorg for another PR.

Reply:
sure, I can just move it to `layers/_tests`